src/renderer/pages/Staking/Overview/Overview.tsx

```diff
@@ -57,6 +58,7 @@ export const Overview = () => {
 
   const accounts = activeAccounts.reduce<Account[]>((acc, account) => {
     if (accountUtils.isChainIdMatch(account, chainId)) {
+      if (accountUtils.isBaseAccount(account) && walletUtils.isPolkadotVault(activeWallet)) return acc;
```

Comment:
Does it need to be nested?
If chain match or not for BaseAccount and PV wallet we will `return acc` anyway

Reply:
For `BaseAccount`  in inChainIdMatch we return true always which is okay if it's not the case for Polkadaot Vault. With this nested condition, it's more readable that we want to skip adding BaseAccount  only if it's PolkadotVault